internal/metamorphic/crossversion/crossversion_test: TestMetaCrossVersion failed

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/18821702?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/18821702?buildTab=artifacts#/) on refs/heads/master @ [9a86078d6ffe](https://github.com/cockroachdb/pebble/commits/9a86078d6ffe):


```
                --- PASS: TestMeta/execution/random-029 (20.58s)
                --- PASS: TestMeta/execution/random-023 (111.33s)
                --- PASS: TestMeta/execution/random-022 (53.48s)
                --- PASS: TestMeta/execution/random-014 (16.59s)
                --- PASS: TestMeta/execution/random-027 (63.60s)
                --- PASS: TestMeta/execution/random-013 (16.11s)
                --- PASS: TestMeta/execution/random-002 (94.38s)
                --- PASS: TestMeta/execution/random-012 (51.51s)
                --- PASS: TestMeta/execution/random-020 (22.88s)
                --- PASS: TestMeta/execution/random-025 (24.56s)
                --- PASS: TestMeta/execution/random-026 (27.73s)
                --- PASS: TestMeta/execution/random-009 (13.79s)
                --- PASS: TestMeta/execution/random-010 (34.96s)
                --- PASS: TestMeta/execution/random-024 (20.74s)
                --- PASS: TestMeta/execution/standard-029 (17.43s)
                --- PASS: TestMeta/execution/standard-028 (19.34s)
                --- PASS: TestMeta/execution/random-028 (123.66s)
                --- PASS: TestMeta/execution/standard-026 (21.64s)
                --- PASS: TestMeta/execution/standard-027 (24.31s)
                --- PASS: TestMeta/execution/standard-025 (21.28s)
                --- PASS: TestMeta/execution/standard-023 (19.15s)
                --- PASS: TestMeta/execution/standard-022 (20.16s)
                --- PASS: TestMeta/execution/standard-024 (22.92s)
                --- PASS: TestMeta/execution/standard-018 (16.03s)
                --- PASS: TestMeta/execution/standard-020 (19.69s)
                --- PASS: TestMeta/execution/standard-019 (19.26s)
                --- PASS: TestMeta/execution/standard-016 (16.19s)
                --- PASS: TestMeta/execution/standard-008 (15.46s)
                --- PASS: TestMeta/execution/standard-017 (16.61s)
                --- PASS: TestMeta/execution/standard-014 (14.81s)
                --- PASS: TestMeta/execution/standard-013 (15.85s)
                --- PASS: TestMeta/execution/standard-012 (17.04s)
                --- PASS: TestMeta/execution/standard-011 (11.00s)
                --- PASS: TestMeta/execution/standard-021 (65.50s)
                --- PASS: TestMeta/execution/standard-010 (10.75s)
                --- PASS: TestMeta/execution/standard-009 (10.61s)
                --- PASS: TestMeta/execution/standard-015 (129.17s)
                --- PASS: TestMeta/execution/standard-007 (7.65s)
                --- PASS: TestMeta/execution/standard-004 (8.50s)
                --- PASS: TestMeta/execution/standard-006 (8.54s)
                --- PASS: TestMeta/execution/standard-005 (7.55s)
                --- PASS: TestMeta/execution/standard-002 (6.87s)
                --- PASS: TestMeta/execution/standard-001 (6.75s)
                --- PASS: TestMeta/execution/standard-003 (7.17s)
                --- FAIL: TestMeta/execution/random-001 (265.95s)
                --- PASS: TestMeta/execution/random-011 (183.77s)
                --- PASS: TestMeta/execution/random-018 (379.43s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=15b2aa8a-seed=2500979814576982890-opts=random-014(<empty>) (2810.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: PEBBLE-341

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/18833128?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/18833128?buildTab=artifacts#/) on refs/heads/master @ [6edd850f8c1a](https://github.com/cockroachdb/pebble/commits/6edd850f8c1a):


```
                --- PASS: TestMeta/execution/standard-010 (10.21s)
                --- PASS: TestMeta/execution/standard-009 (10.81s)
                --- PASS: TestMeta/execution/random-019 (32.14s)
                --- PASS: TestMeta/execution/random-017 (25.02s)
                --- PASS: TestMeta/execution/standard-008 (8.23s)
                --- PASS: TestMeta/execution/random-018 (32.56s)
                --- PASS: TestMeta/execution/random-015 (9.91s)
                --- PASS: TestMeta/execution/random-016 (17.30s)
                --- PASS: TestMeta/execution/standard-007 (8.02s)
                --- PASS: TestMeta/execution/random-014 (8.02s)
                --- PASS: TestMeta/execution/standard-006 (9.19s)
                --- PASS: TestMeta/execution/random-012 (6.05s)
                --- PASS: TestMeta/execution/standard-005 (7.47s)
                --- PASS: TestMeta/execution/standard-004 (9.21s)
                --- PASS: TestMeta/execution/standard-003 (8.48s)
                --- PASS: TestMeta/execution/random-013 (18.98s)
                --- PASS: TestMeta/execution/standard-002 (7.19s)
                --- PASS: TestMeta/execution/random-010 (11.69s)
                --- PASS: TestMeta/execution/standard-001 (8.26s)
                --- PASS: TestMeta/execution/random-006 (8.04s)
                --- PASS: TestMeta/execution/random-011 (26.82s)
                --- PASS: TestMeta/execution/random-007 (15.51s)
                --- PASS: TestMeta/execution/random-008 (11.61s)
                --- PASS: TestMeta/execution/random-002 (7.73s)
                --- PASS: TestMeta/execution/random-003 (10.13s)
                --- PASS: TestMeta/execution/random-004 (10.96s)
                --- PASS: TestMeta/execution/random-001 (15.44s)
                --- PASS: TestMeta/execution/standard-029 (11.67s)
                --- PASS: TestMeta/execution/standard-018 (6.26s)
                --- PASS: TestMeta/execution/random-000 (31.48s)
                --- PASS: TestMeta/execution/standard-020 (8.58s)
                --- PASS: TestMeta/execution/standard-019 (6.96s)
                --- PASS: TestMeta/execution/standard-017 (8.13s)
                --- PASS: TestMeta/execution/standard-016 (8.37s)
                --- PASS: TestMeta/execution/random-025 (41.23s)
                --- PASS: TestMeta/execution/standard-025 (7.16s)
                --- PASS: TestMeta/execution/standard-026 (7.83s)
                --- PASS: TestMeta/execution/standard-027 (8.79s)
                --- PASS: TestMeta/execution/standard-023 (6.90s)
                --- PASS: TestMeta/execution/standard-024 (7.63s)
                --- PASS: TestMeta/execution/standard-022 (6.19s)
                --- PASS: TestMeta/execution/random-005 (103.41s)
                --- PASS: TestMeta/execution/random-028 (305.84s)
                --- PASS: TestMeta/execution/random-023 (415.97s)
                --- PASS: TestMeta/execution/standard-015 (310.96s)
                --- PASS: TestMeta/execution/random-009 (894.35s)
                --- FAIL: TestMeta/execution/random-021 (1005.95s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=15b2aa8a-seed=16962881190164797910-opts=random-021(<empty>) (2692.11s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/18834888?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/18834888?buildTab=artifacts#/) on refs/heads/master @ [6edd850f8c1a](https://github.com/cockroachdb/pebble/commits/6edd850f8c1a):


```
                --- PASS: TestMeta/execution/random-019 (10.85s)
                --- PASS: TestMeta/execution/random-027 (11.62s)
                --- PASS: TestMeta/execution/random-020 (12.57s)
                --- PASS: TestMeta/execution/random-029 (14.78s)
                --- PASS: TestMeta/execution/random-028 (16.96s)
                --- PASS: TestMeta/execution/standard-029 (10.03s)
                --- PASS: TestMeta/execution/standard-028 (6.81s)
                --- PASS: TestMeta/execution/random-021 (30.02s)
                --- PASS: TestMeta/execution/standard-027 (9.47s)
                --- PASS: TestMeta/execution/standard-026 (8.14s)
                --- PASS: TestMeta/execution/standard-025 (8.68s)
                --- PASS: TestMeta/execution/random-013 (74.53s)
                --- PASS: TestMeta/execution/standard-024 (7.50s)
                --- PASS: TestMeta/execution/standard-023 (9.20s)
                --- PASS: TestMeta/execution/standard-022 (8.41s)
                --- PASS: TestMeta/execution/standard-020 (8.80s)
                --- PASS: TestMeta/execution/standard-019 (9.24s)
                --- PASS: TestMeta/execution/standard-018 (9.59s)
                --- PASS: TestMeta/execution/standard-017 (9.69s)
                --- PASS: TestMeta/execution/random-006 (107.56s)
                --- PASS: TestMeta/execution/standard-016 (8.98s)
                --- PASS: TestMeta/execution/random-026 (12.78s)
                --- PASS: TestMeta/execution/standard-021 (29.37s)
                --- PASS: TestMeta/execution/random-025 (18.65s)
                --- PASS: TestMeta/execution/standard-008 (11.26s)
                --- PASS: TestMeta/execution/standard-014 (7.95s)
                --- PASS: TestMeta/execution/standard-013 (8.06s)
                --- PASS: TestMeta/execution/standard-012 (8.09s)
                --- PASS: TestMeta/execution/standard-011 (7.97s)
                --- PASS: TestMeta/execution/standard-010 (9.65s)
                --- PASS: TestMeta/execution/standard-009 (8.76s)
                --- PASS: TestMeta/execution/standard-007 (7.51s)
                --- PASS: TestMeta/execution/standard-004 (8.93s)
                --- PASS: TestMeta/execution/random-007 (31.76s)
                --- PASS: TestMeta/execution/standard-002 (5.27s)
                --- PASS: TestMeta/execution/standard-006 (6.99s)
                --- PASS: TestMeta/execution/standard-005 (7.58s)
                --- PASS: TestMeta/execution/standard-003 (7.61s)
                --- PASS: TestMeta/execution/random-001 (8.35s)
                --- PASS: TestMeta/execution/standard-001 (5.91s)
                --- PASS: TestMeta/execution/random-018 (5.02s)
                --- PASS: TestMeta/execution/random-024 (42.82s)
                --- PASS: TestMeta/execution/random-002 (4.33s)
                --- PASS: TestMeta/execution/random-017 (8.48s)
                --- PASS: TestMeta/execution/random-016 (8.51s)
                --- PASS: TestMeta/execution/standard-015 (86.83s)
                --- FAIL: TestMeta/execution/random-015 (862.12s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=15b2aa8a-seed=3809711021084803414-opts=standard-027(<empty>) (1692.69s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/18912180?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/18912180?buildTab=artifacts#/) on refs/heads/master @ [6949b900b3e1](https://github.com/cockroachdb/pebble/commits/6949b900b3e1):


```
                --- PASS: TestMeta/execution/standard-001 (10.88s)
                --- PASS: TestMeta/execution/standard-025 (12.37s)
                --- PASS: TestMeta/execution/standard-026 (11.64s)
                --- PASS: TestMeta/execution/standard-005 (12.55s)
                --- PASS: TestMeta/execution/standard-017 (9.05s)
                --- PASS: TestMeta/execution/standard-016 (9.66s)
                --- PASS: TestMeta/execution/standard-018 (12.10s)
                --- PASS: TestMeta/execution/random-015 (13.13s)
                --- PASS: TestMeta/execution/random-028 (13.23s)
                --- PASS: TestMeta/execution/standard-021 (33.71s)
                --- PASS: TestMeta/execution/random-000 (50.50s)
                --- PASS: TestMeta/execution/standard-015 (51.46s)
                --- PASS: TestMeta/execution/random-029 (20.39s)
                --- PASS: TestMeta/execution/random-025 (10.05s)
                --- PASS: TestMeta/execution/random-026 (15.20s)
                --- PASS: TestMeta/execution/random-024 (8.67s)
                --- PASS: TestMeta/execution/random-027 (20.21s)
                --- PASS: TestMeta/execution/random-021 (11.07s)
                --- PASS: TestMeta/execution/random-019 (12.32s)
                --- PASS: TestMeta/execution/random-018 (11.45s)
                --- PASS: TestMeta/execution/standard-012 (8.56s)
                --- PASS: TestMeta/execution/standard-014 (9.71s)
                --- PASS: TestMeta/execution/standard-013 (9.65s)
                --- PASS: TestMeta/execution/random-022 (28.32s)
                --- PASS: TestMeta/execution/random-023 (29.03s)
                --- PASS: TestMeta/execution/standard-003 (11.07s)
                --- PASS: TestMeta/execution/random-008 (8.45s)
                --- PASS: TestMeta/execution/random-014 (7.81s)
                --- PASS: TestMeta/execution/standard-024 (9.33s)
                --- PASS: TestMeta/execution/random-009 (16.08s)
                --- PASS: TestMeta/execution/random-017 (46.48s)
                --- PASS: TestMeta/execution/random-020 (50.76s)
                --- PASS: TestMeta/execution/random-013 (28.24s)
                --- PASS: TestMeta/execution/random-005 (10.07s)
                --- PASS: TestMeta/execution/standard-010 (9.35s)
                --- PASS: TestMeta/execution/random-004 (28.81s)
                --- PASS: TestMeta/execution/random-010 (48.63s)
                --- PASS: TestMeta/execution/standard-011 (11.19s)
                --- PASS: TestMeta/execution/random-007 (34.14s)
                --- PASS: TestMeta/execution/standard-009 (9.83s)
                --- PASS: TestMeta/execution/random-012 (60.42s)
                --- PASS: TestMeta/execution/random-003 (19.16s)
                --- PASS: TestMeta/execution/random-002 (38.51s)
                --- PASS: TestMeta/execution/random-011 (251.39s)
                --- PASS: TestMeta/execution/random-016 (281.92s)
                --- FAIL: TestMeta/execution/random-006 (540.94s)
                --- PASS: TestMeta/execution/random-001 (555.01s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=15b2aa8a-seed=4803976618725993710-opts=standard-003(<empty>) (1901.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19028219?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19028219?buildTab=artifacts#/) on refs/heads/master @ [5fb8d04f5328](https://github.com/cockroachdb/pebble/commits/5fb8d04f5328):


```
                --- PASS: TestMeta/execution/standard-026 (11.66s)
                --- PASS: TestMeta/execution/standard-025 (12.48s)
                --- PASS: TestMeta/execution/standard-023 (11.95s)
                --- PASS: TestMeta/execution/standard-024 (13.14s)
                --- PASS: TestMeta/execution/random-009 (32.12s)
                --- PASS: TestMeta/execution/standard-022 (10.84s)
                --- PASS: TestMeta/execution/standard-020 (10.48s)
                --- PASS: TestMeta/execution/standard-019 (11.19s)
                --- PASS: TestMeta/execution/standard-018 (11.08s)
                --- PASS: TestMeta/execution/standard-016 (10.34s)
                --- PASS: TestMeta/execution/standard-017 (12.59s)
                --- PASS: TestMeta/execution/random-021 (10.48s)
                --- PASS: TestMeta/execution/standard-021 (32.24s)
                --- PASS: TestMeta/execution/random-019 (11.00s)
                --- PASS: TestMeta/execution/random-022 (32.03s)
                --- PASS: TestMeta/execution/random-020 (24.01s)
                --- PASS: TestMeta/execution/random-018 (23.71s)
                --- PASS: TestMeta/execution/random-027 (20.24s)
                --- PASS: TestMeta/execution/random-026 (19.37s)
                --- PASS: TestMeta/execution/random-025 (37.81s)
                --- PASS: TestMeta/execution/random-029 (79.39s)
                --- PASS: TestMeta/execution/random-028 (84.66s)
                --- PASS: TestMeta/execution/random-007 (29.53s)
                --- PASS: TestMeta/execution/random-006 (20.80s)
                --- PASS: TestMeta/execution/random-003 (7.23s)
                --- PASS: TestMeta/execution/random-005 (11.73s)
                --- PASS: TestMeta/execution/random-004 (24.12s)
                --- PASS: TestMeta/execution/random-001 (20.99s)
                --- PASS: TestMeta/execution/standard-007 (9.64s)
                --- PASS: TestMeta/execution/standard-013 (10.13s)
                --- PASS: TestMeta/execution/standard-012 (12.23s)
                --- PASS: TestMeta/execution/standard-011 (9.77s)
                --- PASS: TestMeta/execution/standard-010 (8.58s)
                --- PASS: TestMeta/execution/random-002 (46.83s)
                --- PASS: TestMeta/execution/standard-009 (8.25s)
                --- PASS: TestMeta/execution/standard-008 (8.20s)
                --- PASS: TestMeta/execution/standard-004 (9.46s)
                --- PASS: TestMeta/execution/standard-006 (7.62s)
                --- PASS: TestMeta/execution/standard-005 (7.60s)
                --- PASS: TestMeta/execution/standard-002 (6.32s)
                --- PASS: TestMeta/execution/standard-003 (7.15s)
                --- PASS: TestMeta/execution/standard-001 (5.88s)
                --- PASS: TestMeta/execution/random-013 (330.39s)
                --- PASS: TestMeta/execution/standard-015 (794.74s)
                --- FAIL: TestMeta/execution/random-024 (960.41s)
                --- PASS: TestMeta/execution/random-010 (1175.47s)
                --- PASS: TestMeta/execution/random-017 (1388.30s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=15b2aa8a-seed=16237434881526911217-opts=standard-013(<empty>) (1764.75s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19154277?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19154277?buildTab=artifacts#/) on refs/heads/master @ [dbc1c128682f](https://github.com/cockroachdb/pebble/commits/dbc1c128682f):


```
                --- PASS: TestMeta/execution/standard-013 (29.07s)
                --- PASS: TestMeta/execution/standard-011 (27.10s)
                --- PASS: TestMeta/execution/standard-012 (33.42s)
                --- PASS: TestMeta/execution/standard-010 (25.12s)
                --- PASS: TestMeta/execution/standard-009 (27.95s)
                --- PASS: TestMeta/execution/standard-008 (26.27s)
                --- PASS: TestMeta/execution/standard-007 (28.14s)
                --- PASS: TestMeta/execution/standard-006 (29.49s)
                --- PASS: TestMeta/execution/standard-005 (29.30s)
                --- PASS: TestMeta/execution/standard-004 (31.10s)
                --- PASS: TestMeta/execution/standard-003 (28.05s)
                --- PASS: TestMeta/execution/standard-002 (25.87s)
                --- PASS: TestMeta/execution/standard-001 (26.49s)
                --- PASS: TestMeta/execution/random-014 (29.65s)
                --- PASS: TestMeta/execution/random-013 (29.22s)
                --- PASS: TestMeta/execution/random-011 (26.48s)
                --- PASS: TestMeta/execution/random-009 (23.80s)
                --- PASS: TestMeta/execution/random-010 (37.39s)
                --- PASS: TestMeta/execution/random-004 (22.45s)
                --- PASS: TestMeta/execution/random-007 (43.34s)
                --- PASS: TestMeta/execution/random-008 (58.30s)
                --- PASS: TestMeta/execution/random-003 (27.15s)
                --- PASS: TestMeta/execution/random-022 (18.16s)
                --- PASS: TestMeta/execution/random-005 (78.60s)
                --- PASS: TestMeta/execution/random-002 (53.59s)
                --- PASS: TestMeta/execution/random-001 (44.80s)
                --- PASS: TestMeta/execution/random-021 (20.05s)
                --- PASS: TestMeta/execution/random-006 (111.66s)
                --- PASS: TestMeta/execution/random-019 (50.99s)
                --- PASS: TestMeta/execution/standard-022 (19.00s)
                --- PASS: TestMeta/execution/random-020 (99.06s)
                --- PASS: TestMeta/execution/standard-020 (24.37s)
                --- PASS: TestMeta/execution/random-012 (239.14s)
                --- PASS: TestMeta/execution/standard-021 (60.25s)
                --- PASS: TestMeta/execution/standard-019 (19.79s)
                --- PASS: TestMeta/execution/random-018 (131.95s)
                --- PASS: TestMeta/execution/standard-018 (18.48s)
                --- PASS: TestMeta/execution/standard-017 (13.35s)
                --- PASS: TestMeta/execution/standard-016 (13.62s)
                --- PASS: TestMeta/execution/random-024 (18.32s)
                --- FAIL: TestMeta/execution/random-015 (400.12s)
                --- PASS: TestMeta/execution/random-029 (14.15s)
                --- PASS: TestMeta/execution/random-026 (19.32s)
                --- PASS: TestMeta/execution/random-028 (11.29s)
                --- PASS: TestMeta/execution/random-016 (175.75s)
                --- PASS: TestMeta/execution/standard-015 (580.93s)
                --- PASS: TestMeta/execution/random-017 (1823.92s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-master/sha=6667fedd-seed=10616572477033875233-opts=random-018(sha=b416aeef-seed=7447351146675379791-opts=random-001(<empty>)) (2447.90s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19178514?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19178514?buildTab=artifacts#/) on refs/heads/master @ [a45eb8241a67](https://github.com/cockroachdb/pebble/commits/a45eb8241a67):


```
                --- PASS: TestMeta/execution/random-026 (11.57s)
                --- PASS: TestMeta/execution/random-029 (13.31s)
                --- PASS: TestMeta/execution/random-028 (13.94s)
                --- PASS: TestMeta/execution/random-027 (13.69s)
                --- PASS: TestMeta/execution/standard-016 (15.31s)
                --- PASS: TestMeta/execution/random-015 (17.73s)
                --- PASS: TestMeta/execution/random-025 (11.10s)
                --- PASS: TestMeta/execution/standard-021 (45.48s)
                --- PASS: TestMeta/execution/random-024 (19.99s)
                --- PASS: TestMeta/execution/random-022 (22.47s)
                --- PASS: TestMeta/execution/random-019 (17.48s)
                --- PASS: TestMeta/execution/random-018 (14.86s)
                --- PASS: TestMeta/execution/random-017 (18.68s)
                --- PASS: TestMeta/execution/random-020 (37.13s)
                --- PASS: TestMeta/execution/random-014 (17.20s)
                --- PASS: TestMeta/execution/random-012 (17.07s)
                --- PASS: TestMeta/execution/random-016 (50.22s)
                --- PASS: TestMeta/execution/random-008 (53.15s)
                --- PASS: TestMeta/execution/random-011 (32.16s)
                --- PASS: TestMeta/execution/random-010 (26.11s)
                --- PASS: TestMeta/execution/random-021 (86.68s)
                --- PASS: TestMeta/execution/random-009 (15.81s)
                --- PASS: TestMeta/execution/standard-008 (14.09s)
                --- PASS: TestMeta/execution/standard-014 (13.39s)
                --- PASS: TestMeta/execution/standard-013 (13.09s)
                --- PASS: TestMeta/execution/standard-012 (14.23s)
                --- PASS: TestMeta/execution/standard-011 (15.10s)
                --- PASS: TestMeta/execution/standard-010 (12.77s)
                --- PASS: TestMeta/execution/standard-009 (13.74s)
                --- PASS: TestMeta/execution/standard-004 (13.52s)
                --- PASS: TestMeta/execution/standard-007 (12.66s)
                --- PASS: TestMeta/execution/standard-006 (12.62s)
                --- PASS: TestMeta/execution/standard-005 (13.12s)
                --- PASS: TestMeta/execution/standard-002 (11.03s)
                --- PASS: TestMeta/execution/standard-003 (12.43s)
                --- PASS: TestMeta/execution/random-004 (11.97s)
                --- PASS: TestMeta/execution/random-007 (14.81s)
                --- PASS: TestMeta/execution/standard-001 (10.67s)
                --- PASS: TestMeta/execution/random-002 (9.17s)
                --- PASS: TestMeta/execution/random-006 (16.98s)
                --- PASS: TestMeta/execution/random-005 (18.26s)
                --- PASS: TestMeta/execution/random-001 (9.39s)
                --- PASS: TestMeta/execution/random-003 (17.03s)
                --- PASS: TestMeta/execution/random-000 (12.99s)
                --- PASS: TestMeta/execution/random-013 (119.99s)
                --- PASS: TestMeta/execution/standard-015 (374.55s)
                --- FAIL: TestMeta/execution/random-023 (1784.97s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=b416aeef-seed=9388875959073281953-opts=random-000(<empty>) (1960.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19190184?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19190184?buildTab=artifacts#/) on refs/heads/master @ [918c6dd6b02a](https://github.com/cockroachdb/pebble/commits/918c6dd6b02a):


```
                --- PASS: TestMeta/execution/standard-029 (12.68s)
                --- PASS: TestMeta/execution/standard-028 (11.71s)
                --- PASS: TestMeta/execution/random-015 (91.01s)
                --- PASS: TestMeta/execution/standard-027 (16.26s)
                --- PASS: TestMeta/execution/standard-026 (13.44s)
                --- PASS: TestMeta/execution/random-009 (81.06s)
                --- PASS: TestMeta/execution/standard-024 (12.51s)
                --- PASS: TestMeta/execution/standard-023 (11.82s)
                --- PASS: TestMeta/execution/standard-025 (13.60s)
                --- PASS: TestMeta/execution/standard-022 (13.00s)
                --- PASS: TestMeta/execution/standard-015 (133.62s)
                --- PASS: TestMeta/execution/standard-020 (16.15s)
                --- PASS: TestMeta/execution/standard-019 (13.97s)
                --- PASS: TestMeta/execution/standard-018 (14.83s)
                --- PASS: TestMeta/execution/standard-017 (14.71s)
                --- PASS: TestMeta/execution/standard-016 (14.76s)
                --- PASS: TestMeta/execution/standard-021 (46.84s)
                --- PASS: TestMeta/execution/random-020 (19.52s)
                --- PASS: TestMeta/execution/random-025 (111.80s)
                --- PASS: TestMeta/execution/random-029 (152.46s)
                --- PASS: TestMeta/execution/random-017 (12.99s)
                --- PASS: TestMeta/execution/random-018 (19.28s)
                --- PASS: TestMeta/execution/random-004 (8.79s)
                --- PASS: TestMeta/execution/random-005 (11.08s)
                --- PASS: TestMeta/execution/random-007 (18.07s)
                --- PASS: TestMeta/execution/random-027 (170.90s)
                --- PASS: TestMeta/execution/random-006 (34.93s)
                --- PASS: TestMeta/execution/random-003 (19.03s)
                --- PASS: TestMeta/execution/random-002 (22.22s)
                --- PASS: TestMeta/execution/standard-007 (15.22s)
                --- PASS: TestMeta/execution/random-016 (50.67s)
                --- PASS: TestMeta/execution/standard-013 (15.98s)
                --- PASS: TestMeta/execution/standard-011 (16.26s)
                --- PASS: TestMeta/execution/standard-012 (19.95s)
                --- PASS: TestMeta/execution/standard-010 (19.40s)
                --- PASS: TestMeta/execution/standard-009 (16.38s)
                --- PASS: TestMeta/execution/standard-008 (19.28s)
                --- PASS: TestMeta/execution/standard-006 (16.79s)
                --- PASS: TestMeta/execution/standard-004 (20.27s)
                --- PASS: TestMeta/execution/standard-005 (20.22s)
                --- PASS: TestMeta/execution/standard-002 (14.54s)
                --- PASS: TestMeta/execution/standard-003 (13.47s)
                --- PASS: TestMeta/execution/standard-001 (13.10s)
                --- PASS: TestMeta/execution/random-019 (128.25s)
                --- PASS: TestMeta/execution/random-001 (62.03s)
                --- PASS: TestMeta/execution/random-024 (299.22s)
                --- FAIL: TestMeta/execution/random-021 (1131.60s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=b416aeef-seed=16585871052865823354-opts=standard-005(<empty>) (1475.96s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19202683?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19202683?buildTab=artifacts#/) on refs/heads/master @ [96302028f139](https://github.com/cockroachdb/pebble/commits/96302028f139):


```
                --- PASS: TestMeta/execution/standard-022 (4.31s)
                --- PASS: TestMeta/execution/standard-023 (5.53s)
                --- PASS: TestMeta/execution/standard-017 (4.25s)
                --- PASS: TestMeta/execution/standard-019 (5.63s)
                --- PASS: TestMeta/execution/standard-016 (4.54s)
                --- PASS: TestMeta/execution/standard-018 (5.53s)
                --- PASS: TestMeta/execution/standard-014 (5.37s)
                --- PASS: TestMeta/execution/standard-013 (4.78s)
                --- PASS: TestMeta/execution/standard-004 (4.65s)
                --- PASS: TestMeta/execution/standard-005 (6.93s)
                --- PASS: TestMeta/execution/standard-003 (5.33s)
                --- PASS: TestMeta/execution/standard-002 (4.00s)
                --- PASS: TestMeta/execution/standard-001 (5.05s)
                --- PASS: TestMeta/execution/random-000 (27.89s)
                --- PASS: TestMeta/execution/standard-015 (28.41s)
                --- PASS: TestMeta/execution/random-029 (5.14s)
                --- PASS: TestMeta/execution/random-028 (5.66s)
                --- PASS: TestMeta/execution/random-015 (7.57s)
                --- PASS: TestMeta/execution/standard-021 (21.62s)
                --- PASS: TestMeta/execution/random-023 (5.20s)
                --- PASS: TestMeta/execution/random-024 (5.77s)
                --- PASS: TestMeta/execution/random-022 (6.77s)
                --- PASS: TestMeta/execution/random-021 (10.83s)
                --- PASS: TestMeta/execution/random-027 (19.69s)
                --- PASS: TestMeta/execution/random-016 (5.92s)
                --- PASS: TestMeta/execution/random-019 (14.02s)
                --- PASS: TestMeta/execution/standard-010 (5.36s)
                --- PASS: TestMeta/execution/random-018 (16.74s)
                --- PASS: TestMeta/execution/standard-009 (4.90s)
                --- PASS: TestMeta/execution/random-017 (15.87s)
                --- PASS: TestMeta/execution/random-014 (5.98s)
                --- PASS: TestMeta/execution/random-011 (7.70s)
                --- PASS: TestMeta/execution/random-010 (5.73s)
                --- PASS: TestMeta/execution/random-012 (10.53s)
                --- PASS: TestMeta/execution/random-009 (5.45s)
                --- PASS: TestMeta/execution/random-007 (18.26s)
                --- PASS: TestMeta/execution/random-003 (4.88s)
                --- PASS: TestMeta/execution/random-004 (5.19s)
                --- PASS: TestMeta/execution/random-006 (8.93s)
                --- PASS: TestMeta/execution/random-026 (51.42s)
                --- PASS: TestMeta/execution/random-001 (6.12s)
                --- PASS: TestMeta/execution/random-002 (5.87s)
                --- PASS: TestMeta/execution/random-008 (20.45s)
                --- PASS: TestMeta/execution/random-013 (34.85s)
                --- PASS: TestMeta/execution/random-005 (20.44s)
                --- PASS: TestMeta/execution/random-020 (62.09s)
                --- FAIL: TestMeta/execution/random-025 (509.07s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=b416aeef-seed=7913633244064198985-opts=random-023(<empty>) (597.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19255004?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19255004?buildTab=artifacts#/) on refs/heads/master @ [0f1c564b0912](https://github.com/cockroachdb/pebble/commits/0f1c564b0912):


```
                --- PASS: TestMeta/execution/random-006 (38.08s)
                --- PASS: TestMeta/execution/random-002 (33.39s)
                --- PASS: TestMeta/execution/random-011 (11.36s)
                --- PASS: TestMeta/execution/random-029 (25.88s)
                --- PASS: TestMeta/execution/random-026 (13.40s)
                --- PASS: TestMeta/execution/random-022 (13.12s)
                --- PASS: TestMeta/execution/random-019 (15.56s)
                --- PASS: TestMeta/execution/random-003 (56.98s)
                --- PASS: TestMeta/execution/random-021 (11.25s)
                --- PASS: TestMeta/execution/standard-029 (9.98s)
                --- PASS: TestMeta/execution/standard-028 (8.06s)
                --- PASS: TestMeta/execution/standard-027 (10.27s)
                --- PASS: TestMeta/execution/standard-026 (8.79s)
                --- PASS: TestMeta/execution/random-028 (47.53s)
                --- PASS: TestMeta/execution/standard-025 (7.70s)
                --- PASS: TestMeta/execution/standard-024 (8.62s)
                --- PASS: TestMeta/execution/standard-023 (8.37s)
                --- PASS: TestMeta/execution/standard-022 (8.84s)
                --- PASS: TestMeta/execution/standard-020 (7.53s)
                --- PASS: TestMeta/execution/standard-019 (9.16s)
                --- PASS: TestMeta/execution/standard-018 (9.81s)
                --- PASS: TestMeta/execution/standard-017 (9.46s)
                --- PASS: TestMeta/execution/standard-021 (25.07s)
                --- PASS: TestMeta/execution/standard-016 (7.51s)
                --- PASS: TestMeta/execution/random-018 (9.53s)
                --- PASS: TestMeta/execution/standard-008 (10.66s)
                --- PASS: TestMeta/execution/standard-014 (8.41s)
                --- PASS: TestMeta/execution/standard-013 (7.05s)
                --- PASS: TestMeta/execution/standard-012 (6.12s)
                --- PASS: TestMeta/execution/standard-011 (7.80s)
                --- PASS: TestMeta/execution/standard-010 (6.80s)
                --- PASS: TestMeta/execution/standard-009 (7.16s)
                --- PASS: TestMeta/execution/random-013 (8.88s)
                --- PASS: TestMeta/execution/random-020 (100.55s)
                --- PASS: TestMeta/execution/random-016 (8.57s)
                --- PASS: TestMeta/execution/standard-004 (6.49s)
                --- PASS: TestMeta/execution/standard-006 (5.76s)
                --- PASS: TestMeta/execution/standard-007 (6.53s)
                --- PASS: TestMeta/execution/standard-005 (7.11s)
                --- PASS: TestMeta/execution/standard-003 (5.59s)
                --- PASS: TestMeta/execution/standard-002 (6.33s)
                --- PASS: TestMeta/execution/random-017 (56.93s)
                --- PASS: TestMeta/execution/standard-001 (5.06s)
                --- PASS: TestMeta/execution/standard-015 (260.18s)
                --- PASS: TestMeta/execution/random-014 (355.32s)
                --- PASS: TestMeta/execution/random-012 (472.94s)
                --- FAIL: TestMeta/execution/random-009 (1092.84s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=b416aeef-seed=11384712505532625806-opts=random-006(<empty>) (1262.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19382819?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19382819?buildTab=artifacts#/) on refs/heads/master @ [5242eb16bdf9](https://github.com/cockroachdb/pebble/commits/5242eb16bdf9):


```
                --- PASS: TestMeta/execution/random-010 (33.70s)
                --- PASS: TestMeta/execution/random-009 (34.65s)
                --- PASS: TestMeta/execution/random-008 (30.29s)
                --- PASS: TestMeta/execution/random-023 (11.45s)
                --- PASS: TestMeta/execution/random-020 (7.58s)
                --- PASS: TestMeta/execution/random-022 (18.82s)
                --- PASS: TestMeta/execution/random-001 (15.79s)
                --- PASS: TestMeta/execution/random-006 (15.29s)
                --- PASS: TestMeta/execution/random-005 (10.46s)
                --- PASS: TestMeta/execution/random-004 (15.09s)
                --- PASS: TestMeta/execution/random-018 (204.25s)
                --- PASS: TestMeta/execution/random-019 (11.29s)
                --- PASS: TestMeta/execution/standard-027 (9.20s)
                --- PASS: TestMeta/execution/random-002 (48.33s)
                --- PASS: TestMeta/execution/random-014 (194.65s)
                --- PASS: TestMeta/execution/standard-028 (12.78s)
                --- PASS: TestMeta/execution/standard-012 (18.69s)
                --- PASS: TestMeta/execution/random-013 (216.24s)
                --- PASS: TestMeta/execution/standard-023 (17.63s)
                --- PASS: TestMeta/execution/standard-022 (16.42s)
                --- PASS: TestMeta/execution/standard-020 (18.79s)
                --- PASS: TestMeta/execution/standard-019 (16.51s)
                --- PASS: TestMeta/execution/standard-018 (17.32s)
                --- PASS: TestMeta/execution/standard-021 (43.08s)
                --- PASS: TestMeta/execution/standard-017 (17.10s)
                --- PASS: TestMeta/execution/standard-016 (17.34s)
                --- PASS: TestMeta/execution/standard-014 (14.82s)
                --- PASS: TestMeta/execution/standard-013 (17.07s)
                --- PASS: TestMeta/execution/standard-006 (19.09s)
                --- PASS: TestMeta/execution/standard-011 (18.86s)
                --- PASS: TestMeta/execution/standard-010 (18.10s)
                --- PASS: TestMeta/execution/standard-009 (17.09s)
                --- PASS: TestMeta/execution/standard-008 (17.67s)
                --- PASS: TestMeta/execution/standard-007 (17.69s)
                --- PASS: TestMeta/execution/standard-003 (19.08s)
                --- PASS: TestMeta/execution/standard-005 (20.83s)
                --- PASS: TestMeta/execution/standard-004 (17.23s)
                --- PASS: TestMeta/execution/random-012 (355.26s)
                --- PASS: TestMeta/execution/standard-002 (12.84s)
                --- PASS: TestMeta/execution/standard-001 (14.75s)
                --- PASS: TestMeta/execution/standard-026 (15.00s)
                --- PASS: TestMeta/execution/standard-025 (13.35s)
                --- PASS: TestMeta/execution/standard-015 (449.75s)
                --- PASS: TestMeta/execution/random-021 (1008.04s)
                --- PASS: TestMeta/execution/random-028 (1960.79s)
                --- FAIL: TestMeta/execution/random-000 (1825.20s)
                --- PASS: TestMeta/execution/random-003 (2100.19s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-master/sha=0bfd4a26-seed=13175147888801867204-opts=standard-018(sha=431a23b6-seed=5284282516529062567-opts=random-026(<empty>)) (2569.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19395686?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19395686?buildTab=artifacts#/) on refs/heads/master @ [0c0e65f03d49](https://github.com/cockroachdb/pebble/commits/0c0e65f03d49):


```
                --- PASS: TestMeta/execution/random-002 (56.06s)
                --- PASS: TestMeta/execution/random-003 (59.49s)
                --- PASS: TestMeta/execution/random-027 (13.59s)
                --- PASS: TestMeta/execution/random-028 (22.21s)
                --- PASS: TestMeta/execution/random-029 (33.85s)
                --- PASS: TestMeta/execution/random-026 (22.14s)
                --- PASS: TestMeta/execution/random-019 (20.69s)
                --- PASS: TestMeta/execution/random-022 (28.99s)
                --- PASS: TestMeta/execution/random-024 (33.62s)
                --- PASS: TestMeta/execution/random-020 (28.94s)
                --- PASS: TestMeta/execution/standard-008 (20.52s)
                --- PASS: TestMeta/execution/standard-014 (20.52s)
                --- PASS: TestMeta/execution/random-021 (46.46s)
                --- PASS: TestMeta/execution/standard-013 (21.79s)
                --- PASS: TestMeta/execution/standard-011 (19.50s)
                --- PASS: TestMeta/execution/standard-012 (23.63s)
                --- PASS: TestMeta/execution/standard-010 (21.03s)
                --- PASS: TestMeta/execution/standard-009 (18.19s)
                --- PASS: TestMeta/execution/random-018 (15.85s)
                --- PASS: TestMeta/execution/standard-004 (23.09s)
                --- PASS: TestMeta/execution/standard-007 (17.45s)
                --- PASS: TestMeta/execution/standard-006 (19.17s)
                --- PASS: TestMeta/execution/random-001 (17.21s)
                --- PASS: TestMeta/execution/standard-005 (20.69s)
                --- PASS: TestMeta/execution/standard-002 (15.11s)
                --- PASS: TestMeta/execution/random-017 (59.11s)
                --- PASS: TestMeta/execution/standard-003 (17.37s)
                --- PASS: TestMeta/execution/standard-023 (17.12s)
                --- PASS: TestMeta/execution/standard-029 (14.53s)
                --- PASS: TestMeta/execution/standard-028 (12.68s)
                --- PASS: TestMeta/execution/standard-027 (17.19s)
                --- PASS: TestMeta/execution/standard-026 (15.30s)
                --- PASS: TestMeta/execution/standard-025 (12.68s)
                --- PASS: TestMeta/execution/standard-019 (14.58s)
                --- PASS: TestMeta/execution/standard-024 (15.28s)
                --- PASS: TestMeta/execution/standard-022 (12.71s)
                --- PASS: TestMeta/execution/random-025 (187.85s)
                --- PASS: TestMeta/execution/standard-020 (13.37s)
                --- PASS: TestMeta/execution/standard-001 (13.07s)
                --- PASS: TestMeta/execution/standard-017 (12.14s)
                --- PASS: TestMeta/execution/standard-018 (10.73s)
                --- PASS: TestMeta/execution/standard-021 (26.68s)
                --- PASS: TestMeta/execution/standard-016 (11.17s)
                --- PASS: TestMeta/execution/standard-015 (384.98s)
                --- FAIL: TestMeta/execution/random-006 (422.47s)
                --- FAIL: TestMeta/execution/random-016 (861.25s)
                --- PASS: TestMeta/execution/random-013 (1250.00s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=431a23b6-seed=13711554281470308242-opts=random-025(<empty>) (1514.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19410332?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19410332?buildTab=artifacts#/) on refs/heads/master @ [6018d0e430ba](https://github.com/cockroachdb/pebble/commits/6018d0e430ba):


```
                --- PASS: TestMeta/execution/random-014 (16.53s)
                --- PASS: TestMeta/execution/random-012 (8.79s)
                --- PASS: TestMeta/execution/random-020 (28.28s)
                --- PASS: TestMeta/execution/random-013 (10.02s)
                --- PASS: TestMeta/execution/random-024 (42.02s)
                --- PASS: TestMeta/execution/random-010 (11.25s)
                --- PASS: TestMeta/execution/random-009 (10.92s)
                --- PASS: TestMeta/execution/random-007 (7.89s)
                --- PASS: TestMeta/execution/random-008 (13.66s)
                --- PASS: TestMeta/execution/random-004 (8.37s)
                --- PASS: TestMeta/execution/random-006 (13.47s)
                --- PASS: TestMeta/execution/random-011 (32.48s)
                --- PASS: TestMeta/execution/random-002 (11.84s)
                --- PASS: TestMeta/execution/standard-028 (8.59s)
                --- PASS: TestMeta/execution/random-003 (31.92s)
                --- PASS: TestMeta/execution/standard-027 (6.55s)
                --- PASS: TestMeta/execution/standard-025 (8.81s)
                --- PASS: TestMeta/execution/standard-026 (10.12s)
                --- PASS: TestMeta/execution/standard-024 (7.92s)
                --- PASS: TestMeta/execution/standard-023 (8.88s)
                --- PASS: TestMeta/execution/standard-022 (9.82s)
                --- PASS: TestMeta/execution/standard-020 (9.03s)
                --- PASS: TestMeta/execution/standard-019 (8.10s)
                --- PASS: TestMeta/execution/standard-021 (25.52s)
                --- PASS: TestMeta/execution/standard-018 (9.09s)
                --- PASS: TestMeta/execution/standard-017 (9.18s)
                --- PASS: TestMeta/execution/standard-016 (9.27s)
                --- PASS: TestMeta/execution/standard-008 (10.90s)
                --- PASS: TestMeta/execution/standard-014 (7.17s)
                --- PASS: TestMeta/execution/standard-013 (7.35s)
                --- PASS: TestMeta/execution/standard-012 (8.48s)
                --- PASS: TestMeta/execution/standard-011 (6.94s)
                --- PASS: TestMeta/execution/standard-009 (5.86s)
                --- PASS: TestMeta/execution/standard-010 (6.39s)
                --- PASS: TestMeta/execution/random-001 (110.55s)
                --- PASS: TestMeta/execution/standard-007 (4.58s)
                --- PASS: TestMeta/execution/standard-004 (5.00s)
                --- PASS: TestMeta/execution/standard-006 (4.26s)
                --- PASS: TestMeta/execution/standard-002 (4.18s)
                --- PASS: TestMeta/execution/standard-005 (5.08s)
                --- PASS: TestMeta/execution/standard-003 (4.95s)
                --- PASS: TestMeta/execution/standard-001 (4.43s)
                --- PASS: TestMeta/execution/random-017 (240.49s)
                --- PASS: TestMeta/execution/random-005 (643.81s)
                --- PASS: TestMeta/execution/standard-015 (693.37s)
                --- PASS: TestMeta/execution/random-018 (1643.49s)
                --- FAIL: TestMeta/execution/random-015 (2186.38s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-master/sha=0bfd4a26-seed=12552037183240224683-opts=random-003(sha=431a23b6-seed=10638841199468667829-opts=standard-003(<empty>)) (2355.58s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19549915?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19549915?buildTab=artifacts#/) on refs/heads/master @ [b751e8763d7b](https://github.com/cockroachdb/pebble/commits/b751e8763d7b):


```
                --- PASS: TestMeta/execution/random-028 (10.71s)
                --- PASS: TestMeta/execution/random-026 (12.71s)
                --- PASS: TestMeta/execution/random-027 (18.25s)
                --- PASS: TestMeta/execution/random-021 (17.19s)
                --- PASS: TestMeta/execution/random-025 (35.01s)
                --- PASS: TestMeta/execution/random-020 (22.08s)
                --- PASS: TestMeta/execution/standard-004 (13.03s)
                --- PASS: TestMeta/execution/random-023 (34.82s)
                --- PASS: TestMeta/execution/standard-006 (12.12s)
                --- PASS: TestMeta/execution/random-019 (42.84s)
                --- PASS: TestMeta/execution/standard-005 (14.38s)
                --- PASS: TestMeta/execution/random-018 (16.04s)
                --- PASS: TestMeta/execution/standard-002 (11.67s)
                --- PASS: TestMeta/execution/standard-003 (10.49s)
                --- PASS: TestMeta/execution/standard-022 (10.99s)
                --- PASS: TestMeta/execution/random-024 (61.27s)
                --- PASS: TestMeta/execution/standard-027 (11.45s)
                --- PASS: TestMeta/execution/standard-028 (12.68s)
                --- PASS: TestMeta/execution/standard-026 (9.30s)
                --- PASS: TestMeta/execution/random-017 (31.92s)
                --- PASS: TestMeta/execution/standard-025 (11.52s)
                --- PASS: TestMeta/execution/standard-023 (12.43s)
                --- PASS: TestMeta/execution/standard-024 (16.04s)
                --- PASS: TestMeta/execution/random-000 (108.29s)
                --- PASS: TestMeta/execution/standard-019 (12.85s)
                --- PASS: TestMeta/execution/standard-001 (14.29s)
                --- PASS: TestMeta/execution/standard-020 (14.19s)
                --- PASS: TestMeta/execution/random-008 (20.61s)
                --- PASS: TestMeta/execution/random-011 (19.32s)
                --- PASS: TestMeta/execution/random-012 (20.85s)
                --- PASS: TestMeta/execution/standard-021 (43.07s)
                --- PASS: TestMeta/execution/random-009 (17.97s)
                --- PASS: TestMeta/execution/random-010 (35.21s)
                --- PASS: TestMeta/execution/random-007 (16.17s)
                --- PASS: TestMeta/execution/random-004 (37.27s)
                --- PASS: TestMeta/execution/random-002 (11.32s)
                --- PASS: TestMeta/execution/random-006 (23.48s)
                --- PASS: TestMeta/execution/standard-018 (10.52s)
                --- PASS: TestMeta/execution/random-013 (70.91s)
                --- PASS: TestMeta/execution/standard-017 (12.58s)
                --- PASS: TestMeta/execution/random-005 (34.70s)
                --- PASS: TestMeta/execution/random-003 (22.86s)
                --- PASS: TestMeta/execution/standard-016 (12.61s)
                --- PASS: TestMeta/execution/random-016 (68.23s)
                --- PASS: TestMeta/execution/random-014 (105.89s)
                --- PASS: TestMeta/execution/standard-015 (234.76s)
                --- FAIL: TestMeta/execution/random-001 (935.72s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=57059d65-seed=12080449389036907353-opts=standard-023(<empty>) (1244.50s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19554884?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlyPebbleMetamorphicCrossversion/19554884?buildTab=artifacts#/) on refs/heads/master @ [b751e8763d7b](https://github.com/cockroachdb/pebble/commits/b751e8763d7b):


```
                --- PASS: TestMeta/execution/standard-012 (9.13s)
                --- PASS: TestMeta/execution/standard-015 (314.37s)
                --- PASS: TestMeta/execution/random-001 (11.68s)
                --- PASS: TestMeta/execution/standard-004 (9.66s)
                --- PASS: TestMeta/execution/standard-007 (8.18s)
                --- PASS: TestMeta/execution/standard-006 (7.85s)
                --- PASS: TestMeta/execution/standard-005 (10.25s)
                --- PASS: TestMeta/execution/standard-010 (9.64s)
                --- PASS: TestMeta/execution/standard-002 (7.54s)
                --- PASS: TestMeta/execution/standard-003 (8.74s)
                --- PASS: TestMeta/execution/standard-009 (10.32s)
                --- PASS: TestMeta/execution/standard-001 (8.47s)
                --- PASS: TestMeta/execution/standard-011 (8.04s)
                --- PASS: TestMeta/execution/random-002 (140.99s)
                --- PASS: TestMeta/execution/random-026 (453.79s)
                --- PASS: TestMeta/execution/random-020 (441.59s)
                --- PASS: TestMeta/execution/random-012 (518.19s)
                --- FAIL: TestMeta/execution/random-013 (1445.88s)
                --- FAIL: TestMeta/execution/random-000 (1723.83s)
                --- PASS: TestMeta/execution/random-006 (1818.84s)
        FAIL
        
        --- FAIL: TestMetaCrossVersion/execution-crl-release-24.3/sha=57059d65-seed=2122131461082940690-opts=random-002(<empty>) (2188.06s)
=== RUN   TestMetaCrossVersion/execution-crl-release-24.3/sha=57059d65-seed=2122131461082940690-opts=random-029(<empty>)
=== PAUSE TestMetaCrossVersion/execution-crl-release-24.3/sha=57059d65-seed=2122131461082940690-opts=random-029(<empty>)
=== CONT  TestMetaCrossVersion/execution-crl-release-24.3/sha=57059d65-seed=2122131461082940690-opts=random-029(<empty>)
    crossversion_test.go:283:   Running test with version 1c5cb7f1 with initial state sha=57059d65-seed=2122131461082940690-opts=random-029(<empty>).
        --- PASS: TestMetaCrossVersion/execution-crl-release-24.3/sha=57059d65-seed=2122131461082940690-opts=random-029(<empty>) (643.11s)
panic: incomplete test history; this shouldn't happen given that execution completed successfully [recovered]
	panic: incomplete test history; this shouldn't happen given that execution completed successfully

goroutine 10 [running]:
testing.tRunner.func1.2({0xb4ac60, 0xd29e60})
	GOROOT/src/testing/testing.go:1632 +0x230
testing.tRunner.func1()
	GOROOT/src/testing/testing.go:1635 +0x35e
panic({0xb4ac60?, 0xd29e60?})
	GOROOT/src/runtime/panic.go:791 +0x132
github.com/cockroachdb/pebble/metamorphic.reorderHistory({0xc00059c000, 0x128e, 0xc001424000?})
	external/com_github_cockroachdb_pebble/metamorphic/history.go:212 +0x119
github.com/cockroachdb/pebble/metamorphic.CompareHistories({0xd30160, 0xc000248000}, {0xc000060a08, 0x21c, 0x7ffdc9208788?})
	external/com_github_cockroachdb_pebble/metamorphic/history.go:181 +0x72
internal/metamorphic/crossversion/crossversion_test.runCrossVersion({0xd31e20, 0x11827a0}, 0xc000248000, {0xc0000385a0, 0x56}, {0xc0001be780, 0x5, 0x604?}, 0x183a62d5a7eb5175, 0xa)
	external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:177 +0x405
internal/metamorphic/crossversion/crossversion_test.TestMetaCrossVersion(0xc000248000)
	external/com_github_cockroachdb_pebble/internal/metamorphic/crossversion/crossversion_test.go:125 +0x554
testing.tRunner(0xc000248000, 0xc977c0)
	GOROOT/src/testing/testing.go:1690 +0xf4
created by testing.(*T).Run in goroutine 1
	GOROOT/src/testing/testing.go:1743 +0x390
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMetaCrossVersion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/metamorphic/options.go
+++ b/metamorphic/options.go
@@ -322,8 +322,7 @@ func defaultOptions(kf KeyFormat) *pebble.Options {
 	opts := &pebble.Options{
 		// Use an archive cleaner to ease post-mortem debugging.
 		Cleaner: base.ArchiveCleaner{},
-		// Always use our custom comparer which provides a Split method,
-		// splitting keys at the trailing '@'.
+		// Always use our custom comparer which provides a Split method.
 		Comparer:           kf.Comparer,
 		KeySchema:          kf.KeySchema.Name,
 		KeySchemas:         sstable.MakeKeySchemas(kf.KeySchema),
@@ -336,27 +335,34 @@ func defaultOptions(kf KeyFormat) *pebble.Options {
 	}
 	opts.Experimental.EnableColumnarBlocks = func() bool { return true }
 
-	// We don't want to run the level checker every time because it can slow down
-	// downloads and background compactions too much.
+	// The level checker runs every time a new read state is installed: every
+	// compaction, flush, ingest completion, etc. It runs while the database
+	// mutex DB.mu is held, preventing the scheduling of new compactions or
+	// flushes.
 	//
-	// We aim to run it once every 500ms (on average). To do this with some
-	// randomization, each time we get a callback we see how much time passed
-	// since the last call and run the check with a proportional probability.
-	const meanTimeBetweenChecks = 500 * time.Millisecond
+	// We only consider running the level checker 50% of the time (to ensure
+	// we're not obscuring races post-read state installation).
+	//
+	// Additionally, some option configurations can create pathological numbers
+	// of sstables, causing the level checker to consume an excessive amount of
+	// time. To prevent pathological cases, we limit the cumulative time spent
+	// in the level checker to 25% of the test's runtime. If DebugCheck is
+	// invoked but we've spent more than 25% of our total test time wihtin the
+	// level checker, we skip invoking DebugCheckLevels.
 	startTime := time.Now()
-	// lastCallTime stores the time of the last DebugCheck call, as the duration
-	// since startTime.
-	var lastCallTime atomic.Uint64
+	var cumulativeTime atomic.Int64
 	opts.DebugCheck = func(db *pebble.DB) error {
-		now := time.Since(startTime)
-		last := time.Duration(lastCallTime.Swap(uint64(now)))
-		// Run the check with probability equal to the time (as a fraction of
-		// meanTimeBetweenChecks) passed since the last time we had a chance, as a
-		// fraction of meanTimeBetweenChecks.
-		if rand.Float64() < float64(now-last)/float64(meanTimeBetweenChecks) {
-			return pebble.DebugCheckLevels(db)
+		if rand.Float64() < 0.50 {
+			return nil
 		}
-		return nil
+		now := time.Now()
+		testDur := now.Sub(startTime)
+		if checkerDur := time.Duration(cumulativeTime.Load()); checkerDur > testDur/4 {
+			return nil
+		}
+		err := pebble.DebugCheckLevels(db)
+		cumulativeTime.Add(int64(time.Since(now)))
+		return err
 	}
 
 	return opts

